jubilant/statustypes.py

```diff
@@ -742,3 +744,11 @@ def _from_dict(cls, d: dict[str, Any]) -> Status:
                 else ControllerStatus()
             ),
         )
+
+    def __repr__(self):
+        """Return a pretty-printed version of the status."""
+        return _pretty._dump(self)
+
+    def __str__(self):
+        """Return a pretty-printed version of the status."""
+        return repr(self)
```

Comment:
I'm not sure about this one, but don't have a strong opinion.

My concern is that pretty-print is multiline, while str format should be concise, so ideally not span multiple lines.